code/game/jobs/job/civilians/support/working_joe.dm

```diff
@@ -1,7 +1,7 @@
 /datum/job/civilian/working_joe
 	title = JOB_WORKING_JOE
-	total_positions = 8
-	spawn_positions = 8
+	total_positions = 3
+	spawn_positions = 3
```

Comment:
```suggestion
	total_positions = 8
	spawn_positions = 8
```
Not sure why you're decreasing max slots. Currently it's never hit that amount, and you've not documented it in Changelog.

Reply:
I asked for the total to be lowered. Just add it to the changelog.

Reply:
I've added it, forgot about it previously. Should be good now